bash/force-level1-csd.sh

```diff
@@ -58,25 +57,26 @@ Mandatory arguments:
 
   queue
   Downloaded files are appended to a file queue, which is needed for the 
-  Level 2 processing. The file doesn't need to exist. If it exists, new 
-  lines will be appended on successful ingestion
+  Level 2 processing. The file doesn't need to exist. If it does exist,
+  new lines will be appended on successful ingestion
 
   area of interest
   (1) user-supplied coordinates of your study area: 
       The polygon must be closed (first X/Y = last X/Y). X/Y must be given as
       decimal degrees with negative values for West and South coordinates.
-      You can either give the path to a file, or give the coordinates on the command line.
-      If in the file, put one coordinate per line.
-      If on the command line, give a comma separated list.
+      Either specify the path to a file, or the coordinates on the command line.
+      If in a file, provide one coordinate per line.
+      If on the command line, provide a comma separated list.
   (2) a shapefile (point/polygon/line). On-the-fly reprojection is provided,
       but using EPSG4326 is recommended.
-  (3) Scene identifier.
```

Comment:
IMO, scene would be the correct term. Grid implies sth regular with fixed spacing between observations.

Reply:
I agree that grid is not the perfect term. However, scene identifier is misleading, as this term is used to uniquely identify a certain scene (i.e., LT51350412007264BJC00). 
I noticed this when writing the tutorial, where using 'scene identifier' would really have been ambiguous. Any other ideas? I haven't come up with something I'm really happy.

Reply:
Actually, scene identifies the Path/Row. It is misused frequently.

Reply:
Hmm okay. 
So if we include the search for specific acquisitions - would we call that acquisition identifier?

Reply:
image ID? IDK, I mean, many people use scene here. It is not correct though

Reply:
what about "reference system tile ID" here?